test/unit/definitions/divorce/AuthorisationCaseField.test.js

```diff
@@ -16,24 +16,26 @@ describe('AuthorisationCaseField', () => {
     before(() => {
       nonProdAuthorisationCaseField = getAuthorisationCaseFieldDefinitions([
         'AuthorisationCaseField',
-        'AuthorisationCaseField-amend-court-orders-nonprod',
-        'AuthorisationCaseField-alternative-service-nonprod',
         'AuthorisationCaseField-alt-service-process-server-nonprod',
+        'AuthorisationCaseField-alternative-service-nonprod',
+        'AuthorisationCaseField-amend-court-orders-nonprod',
         'AuthorisationCaseField-deemed-and-dispensed-nonprod',
         'AuthorisationCaseField-general-email-nonprod',
         'AuthorisationCaseField-general-referral-nonprod',
-        'AuthorisationCaseField-share-a-case-nonprod',
-        'AuthorisationCaseField-nonprod'
+        'AuthorisationCaseField-nonprod',
```

Comment:
Personal taste: Can you leave "non-prod" in the bottom of the list (as that's the one that won't be removed in releases)?

Reply:
or on top. just below the core file.

Reply:
Im a fan of that too, however @yarpo wanted it to be alphabetised. But im happy to move forwards with it alphabetised with the exception of keeping default non prod at the top. How does that sound